pkg/executor/sortexec/parallel_sort_worker.go

```diff
@@ -73,9 +72,9 @@ func newParallelSortWorker(
 		timesOfRowCompare:      0,
 		memTracker:             memTracker,
 		sortedRowsIter:         sortedRowsIter,
-		maxSortedRowsLimit:     maxSortedRowsLimit,
+		maxSortedRowsLimit:     maxChunkSize * 30,
 		spillHelper:            spillHelper,
-		batchRows:              make([]chunk.Row, 0, maxSortedRowsLimit),
+		batchRows:              make([]chunk.Row, 0),
```

Comment:
Can you add some benchmarks for it?

Reply:
> Can you add some benchmarks for it?

parallel sort benchmark?

Reply:
Yes, BTW can you check ```BenchmarkUnionScanTableReadDescRead```， ```BenchmarkUnionScanIndexReadDescRead``` and ```BenchmarkUnionScanIndexLookUpDescRead``` wich can test for parallel sort?

Reply:
> Yes, BTW can you check `BenchmarkUnionScanTableReadDescRead`， `BenchmarkUnionScanIndexReadDescRead` and `BenchmarkUnionScanIndexLookUpDescRead` wich can test for parallel sort?

okk

Reply:
> Yes, BTW can you check `BenchmarkUnionScanTableReadDescRead`， `BenchmarkUnionScanIndexReadDescRead` and `BenchmarkUnionScanIndexLookUpDescRead` wich can test for parallel sort?

fixed:
BenchmarkUnionScanTableReadDescRead:
1863      618281 ns/op    148918 B/op     2677 allocs/op
BenchmarkUnionScanIndexReadDescRead:
1922      620293 ns/op    153910 B/op     2750 allocs/op
BenchmarkUnionScanIndexLookUpDescRead:
1744      673479 ns/op    248234 B/op     2820 allocs/op

master:
BenchmarkUnionScanTableReadDescRead:
1826      599392 ns/op    215502 B/op     2674 allocs/op
BenchmarkUnionScanIndexReadDescRead:
1964      593482 ns/op    219032 B/op     2740 allocs/op
BenchmarkUnionScanIndexLookUpDescRead:
1791      659653 ns/op    320267 B/op     2815 allocs/op

dataset: tpch10
sql1: explain analyze select L_COMMENT, L_EXTENDEDPRICE from lineitem where L_SUPPKEY > 95000 order by L_COMMENT desc, L_EXTENDEDPRICE asc;

sql2: explain analyze select * from lineitem where L_SUPPKEY > 95000 order by L_COMMENT desc, L_EXTENDEDPRICE asc;

  | Sql1 | Sql2
-- | -- | --
Master | 7.30s | 15.36s
Fixed | 6.94s | 16.36s

Reply:
add it to PR's description.

Reply:
> add it to PR's description.

done